samples/bluetooth/unicast_audio_client/boards/nrf5340_audio_dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,8 @@
+# For LC3 the following configs are needed
+CONFIG_FPU=y
```

Comment:
`LIBLC3` already depends on `FPU`. Additionally, would it make sense to have it depend on `REQUIRES_FULL_LIBC` and get rid of `CONFIG_NEWLIB_LIBC=y` below?